src/Uno.UI/Microsoft/UI/Xaml/Controls/Materials/Backdrop/BackdropMaterial.cs

```diff
@@ -55,8 +55,8 @@ private static void CreateOrDestroyMicaController()
 			// Or if we're disconnecting the last one, clean up the shared MicaController.
 			if (_connectedBrushCount.Value > 0 && _micaController.Value == null)
 			{
-				var currentWindow = Window.Current;
-
+				// TODO Uno: Adjust this based on updated WinUI3 sources
```

Comment:
Should we open an issue to track? Or augment an existing one?

Reply:
The API is apparently gone from WinUI 3 and fully replaced by `SystemBackdrop` API https://github.com/unoplatform/uno/issues/14853. Will update the comment accordingly